ArduPlane/quadplane.cpp

```diff
@@ -1566,6 +1566,7 @@ void SLT_Transition::update()
     // the tilt will decrease rapidly)
     if (quadplane.tiltrotor.fully_fwd() && transition_state != TRANSITION_AIRSPEED_WAIT) {
         if (transition_state == TRANSITION_TIMER) {
+            plane.TECS_controller.reset();
```

Comment:
Plane pitch control can be coming from TECS at this point (depending on mode). Because we re-set TECS to the current pitch attitude this might cause a twitch. We should reset to the current pitch target.

Reply:
Agree. I will add a separate end of transition reset method.